axlearn/common/attention.py

```diff
@@ -2943,6 +2960,7 @@ def _forward_for_mode(
         self_attention_logit_biases: Optional[Tensor] = None,
         cross_attention_data: Optional[Tensor] = None,
         cross_attention_logit_biases: Optional[Tensor] = None,
+        segment_ids: Optional[Tensor] = None,
```

Comment:
Have we considered calling this e.g. `target_segment_ids` to disambiguate with potential cross attention?

Reply:
Good point - renamed to `target_segment_ids`.